### PR TITLE
Added the static versions in package.json instead of wildcard *

### PR DESCRIPTION
**Pull request by gr8pathik**
I have added the static version in package.json to solve the issue mention in #10. Please accept this PR as soon as possible so people can run this app on local and help to improve this package.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,21 @@
...
+# misc
+.DS_Store
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+.idea
+.storybook/build/storybook_static
+
```
Comment (phansys) [inline .gitignore:21]: Please, remove extra LF at EOF.

File: .gitignore [hunk 2]
```diff
@@ -1 +1,21 @@
...
+/node_modules
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```
Comment (phansys) [inline .gitignore:13]: Please, don't add environment specific rules to this file. Use [global ignoring rules](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) on your environment instead.

File: package-lock.json [hunk 3]
```diff
@@ -0,0 +1,15279 @@
+{
```
Comment (phansys) [inline package-lock.json:1]: This is a library, not a project. Why is this file required under the SCM?

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs work.

**Comment by one-more** [comment]
fixed in https://github.com/one-more/react-cron-builder/commit/a50928a20cc4b4144858bbdb7a035f2aff244245

**Pull request closed without merging**